server/jms/src/main/java/org/hawkular/apm/server/jms/CommunicationDetailsCacheMDB.java

```diff
@@ -30,17 +30,20 @@
 import com.fasterxml.jackson.core.type.TypeReference;
 
 /**
+ * This MDB is provided to populate the CommunicationDetailsCache when Communication Details.
+ * Each clustered APM server node will receive the communication details data, so each cache
+ * is expected to only be a local cache.
+ *
+ * Discussion regarding potential future use of a distributed cache is associated with HWKAPM-479.
+ * If this change happens, then this MDB should be converted back to a durable (load balanced) subscriber.
+ *
  * @author gbrown
  */
-@MessageDriven(name = "CommunicationDetails_Cache", messageListenerInterface = MessageListener.class,
+@MessageDriven(name = "CommunicationDetails_CommunicationDetailsCache", messageListenerInterface = MessageListener.class,
         activationConfig =
         {
                 @ActivationConfigProperty(propertyName = "destinationType", propertyValue = "javax.jms.Topic"),
-                @ActivationConfigProperty(propertyName = "destination", propertyValue = "CommunicationDetails"),
-                @ActivationConfigProperty(propertyName = "subscriptionDurability", propertyValue = "Durable"),
```

Comment:
This is specific behaviour that I required, that is supported by Artemis, where if more than one durable subscribers have the same subscription name and client id, then they only one will receive the message - it is essentially providing load balancing queue behaviour but when subscribing to a topic - defined by the consumer - so topic publisher is oblivious of how the message is being consumed - which also means normal non durable and durable subscribers can receive receive the message on multiple nodes - so the behaviour you would normally expect from a topic subscriber.